src/azure-cli-core/azure/cli/core/local_context.py

```diff
@@ -133,17 +133,17 @@ def delete_file(self, recursive=False):
                 parent_dir = os.path.dirname(local_context_file.config_path)
                 if not os.listdir(parent_dir):
                     shutil.rmtree(parent_dir)
-                logger.warning('Local context persistence file in working directory %s is deleted.',
```

Comment:
How about we change the file name `local_context.py` to avoid inconsistency?

Reply:
Just as I described in the PR description, this PR is only attempted to update the UE part, actually the internal class / persist file / test framework are still called `Local Context` (az local-context is marked deprecated but not totally removed yet). Let's leave the internal renaming work to totally remove `az local-context` command.